### ISSUE TITLE
Centcom ferry fuckery

### ISSUE THREAD

**Issue by DaxDupont** [tgstation/tgstation#36773]
Issue reported from  Round ID: 85679 (/tg/Station Bagil [ENGLISH] [US-EAST] [100% FREE LAG])

Reporting client version: 512

Damaged Centcom ferry's rebuilt parts stayed behind.

Since @ninjanomnom can't reproduce it i am guessing its not a problem with his testmerge.

**Comment by ninjanomnom** [comment]
This should be two separate issues.

Also seriously, how do you expect me to be able to replicate issues with no details. The rebuilt parts being left behind may well be the testmerged pr but you need to give details if you hope to get things figured out.

**Comment by ninjanomnom** [comment]
What shuttle didn't get repaired? What turf type was it? What method of repair did you use?

Explain *all* details in issue reports so we don't have to waste time checking various ways a problem could happen.

**Comment by DaxDupont** [comment]
It was the centcom ferry, that's literally it's name. It's the one ERTs and DS uses.

Both manually rebuild walls using RCD and parts foamed with smart foam were left behind.
So literally all the rebuilt parts in both methods on the thing that is name centcom ferry.

I don't know what fucked it in the first place but I think it was a kaboom.

tl;dr rebuilt parts on centcom ferry get left behind.

**Comment by DaxDupont** [comment]
Also I split these up now.

**Comment by ninjanomnom** [comment]
I believe this has been fixed by a commit in the testmerged pr. Not all plating subtypes counted when repairing shuttles.
